prow/scripts/cluster-integration/kyma-gke-nightly.sh

```diff
@@ -186,55 +209,26 @@ function installStabilityChecker() {
 	        --name=stability-checker
 }
 
-readonly REPO_OWNER="kyma-project"
-export REPO_OWNER
-readonly REPO_NAME="kyma"
-export REPO_NAME
-
-export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
-export KYMA_SOURCES_DIR="${KYMA_PROJECT_DIR}/kyma"
-export KYMA_SCRIPTS_DIR="${KYMA_SOURCES_DIR}/installation/scripts"
-export GCLOUD_SERVICE_KEY_PATH="${GOOGLE_APPLICATION_CREDENTIALS}"
-
-readonly CURRENT_TIMESTAMP=$(date +%Y%m%d)
-
-if [[ "$OSTYPE" == "darwin"* ]]; then
-	readonly LAST_TIMESTAMP=$(date -v -1d '+%Y%m%d')
-else
-	readonly LAST_TIMESTAMP=$(date +%Y%m%d --date="yesterday")
-fi
-
-readonly NAME_ROOT="gkeint-nightly"
-readonly COMMON_NAME=$(echo "${NAME_ROOT}-${CURRENT_TIMESTAMP}" | tr "[:upper:]" "[:lower:]")
-export COMMON_NAME
-readonly KYMA_INSTALLER_IMAGE="${DOCKER_PUSH_REPOSITORY}${DOCKER_PUSH_DIRECTORY}/${NAME_ROOT}/${REPO_OWNER}/${REPO_NAME}:${CURRENT_TIMESTAMP}"
-export KYMA_INSTALLER_IMAGE
-
-### Cluster name must be less than 40 characters!
-export CLUSTER_NAME="${COMMON_NAME}"
-
-### For provision-gke-cluster.sh
-export GCLOUD_PROJECT_NAME="${CLOUDSDK_CORE_PROJECT}"
-export GCLOUD_COMPUTE_ZONE="${CLOUDSDK_COMPUTE_ZONE}"
+function cleanup() {
+    OLD_CLUSTERS=$(gcloud container clusters list --filter="name~${NAME_ROOT}" --format json | jq '.[].name' | tr -d '"')
```

Comment:
Be more specific about the name. The pattern should be: `^${NAME_ROOT}-`. Verify if it works manually before merging.